From 1ba6822eecbd4d0df1fe88b713ff52b7effea5fb Mon Sep 17 00:00:00 2001
From: Christian Hewitt <christianshewitt@gmail.com>
Date: Mon, 29 Jun 2020 15:54:45 +0000
Subject: [PATCH 122/152] WIP: dt-bindings: arm: amlogic: add A95XF3-AIR
 bindings

Add the board bindings for the CYX A95XF3-AIR STB device.

Signed-off-by: Christian Hewitt <christianshewitt@gmail.com>
---
 Documentation/devicetree/bindings/arm/amlogic.yaml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/arm/amlogic.yaml b/Documentation/devicetree/bindings/arm/amlogic.yaml
index d0bb581ee9ee..55605571e182 100644
--- a/Documentation/devicetree/bindings/arm/amlogic.yaml
+++ b/Documentation/devicetree/bindings/arm/amlogic.yaml
@@ -164,6 +164,7 @@ properties:
           - enum:
               - amediatech,x96-air-2g
               - amediatech,x96-air-4g
+              - cyx,a95xf3-air
               - hardkernel,odroid-c4
               - khadas,vim3l
               - seirobotics,sei610
-- 
2.17.1

